tools/applicationinsights-web-snippet/src/snippet.ts

```diff
@@ -206,12 +207,24 @@ declare var cfg:ISnippetConfig;
                 });
                 // let message = "Load Version 2 SDK instead to support IE"; // where to report this error?
             }
+            if (appInsights.cr){
+                for (var i = 0; i < domains.length; i++){
+                    if (targetSrc.indexOf(domains[i])){
```

Comment:
This will always be true even if not match ;-)  as returning -1 (no match) will still be true.